apps/explorer/templates/explorer/pixelset_list.html

```diff
@@ -146,6 +149,29 @@ <h4>{% trans "Filters" %}</h4>
 {% block javascript %}
 <script>
   (function (document) {
+    // Clean pixelset-filters-form
+    $cleanButton = document.querySelector('button.action-clean');
```

Comment:
No need for the `button` in the selector. You can add an `id` to that button if you want to make sure it is unique.